pkg/controller/ads/ads_controller.go

```diff
@@ -64,7 +64,7 @@ func (c *Controller) AdsStreamCreateAndSend(client service_discovery_v3.Aggregat
 		stopCh:       make(chan struct{}),
 	}
 
-	c.Processor.Reset()
+	c.Processor = newProcessor()
```

Comment:
We donot want to start from zero. So donot init a totally new processor

Reply:
I perform the operation of resetting the Cache in Reset now